README.md

```diff
@@ -95,6 +96,16 @@ let toggleQuality = (function() {
 })();
 
 let currentSelectedQualityLevelIndex = qualityLevels.selectedIndex; // -1 if no level selected
+
+// Listen to change events for when the player selects a new quality level
+// NOTE: Just like populating the list, it is up to the consumer of this plugin to
```

Comment:
It may be helpful to further define consumer of the plugin (to establish the difference between a user and someone implementing a playback tech that uses it).